lib/filter_lib/src/query_modificator.py

```diff
@@ -70,7 +70,10 @@ def form_query(
         pagination = Pagination(page_num=1, page_size=15).dict()
     try:
         query, pag = make_pagination(
-            query, pagination["page_num"], pagination["page_size"]
+            query,
+            pagination.get("page_num"),
```

Comment:
Unify access to keys: use `[...]` or `.get("...")`

Reply:
unified to using .get()